EIPS/eip-2387.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: 2387
+title: "Hardfork Meta: Mountain Glacier"
+author: James Hancock (@madeoftin)
+discussions-to: https://github.com/ethereum/EIPs/pull/2387
```

Comment:
Discussion URL cannot be a pull request according to EIP-1.

Reply:
just making an Ethereum Magicans Thread